src/includes/before-breadcrumb/flutter.mdx

```diff
@@ -0,0 +1,9 @@
+```dart
+  import 'package:sentry/sentry.dart';
+
+  Breadcrumb beforeBreadCrumb(Breadcrumb crumb, dynamic hint) {
+    return 'a.spammy.Logger' == crumb.category ? null : crumb;
+  }
+
+  Sentry.init((options) => options.beforeBreadcrumb = beforeBreadCrumb);
```

Comment:
using internal methods because we support min dart sdk 2.0
and sometimes we have some examples that a `typedef function` would need more than 1 block of code like the example below (breadcrumbs-example) and this will complain https://dart.dev/tools/diagnostic-messages#sdk_version_set_literal

so I've decided to do the examples like this, so we dont need to upgrade the min sdk only because of that, although people probably know that they can inline this if using newer sdk versions.

Reply:
Makes sense for now. But I honestly think it's overkill to target Dart 2.0.0. It's just too old and Dart isn't Java, people upgrade more often.

The oldest version [Flutter lists on their website, 1.12.13](https://flutter.dev/docs/development/tools/sdk/release-notes) uses **Dart 2.7** and is almost a year old (which in Flutter time is forever ago).

Reply:
the version set literals require Dart 2.3 iirc